src/main/java/sirius/biz/process/ProcessContext.java

```diff
@@ -41,6 +42,7 @@
  * install this using {@link sirius.kernel.async.TaskContext#setAdapter(TaskContextAdapter)} so that calls to
  * {@link sirius.kernel.async.TaskContext} will be delegated to the processes framework.
  */
+@ThreadSafe
```

Comment:
Possible RaceConditions via RateLimiting #check
... however, it is probably not too important whether 1-2 additional rate-limited messages are logged or not

Reply:
Good catch! Maybe we should fix this in RateLimit...